lessons/05-css-development-grace/README.md

```diff
@@ -194,126 +194,68 @@ With this property set, the `width` and `height` properties set the dimensions o
-Install the dependencies:
+Speaking of specificity, CSS also supports **nesting**, which allos you to change the appearance of elements based on their nesting properties in the HTML.  For example, in this situation:
 
-```bash
-npm install express morgan body-parser cookie-parser
+```css
+<a href="#">link</a>
+<p>This is a paragraph with <a href="#">link</a></p>
 ```
 
-Run the server and visit the page (you will probably have to refresh because on your first visit you don't have any cookies) to see your cookies:
-
-```bash
-{ 'connect.sid': 's:oJzNzHd1WX-xlEBtpFoy7VoP4UzoqTQ8.VjMIMfdIZqI0oLGy1zh5JwWPbdost5USN9jFYeL9ITY' }
+```css
+a {
+background-color: red
+}
+a:hover {
+background-color: green;
+}
+p a { 
+background-color: green; 
+}
+p a:hover {
+background-color: red;
+}
 ```
 
-As you can see, a cookie is a key-value store that our cookier parser has converted to JSON.
-This is probably a unique ID that we can use to associate a cookie with other data stored on the client.
-
-## Sessions
+What this CSS does is specify that links have a red background that becomes green on hover.  However, the last two definitions override that with specify  - a link that is nested within a `p` element is the opposite - green, and red on hover.
 
-One of the most common ways to use cookies is to keep track of "sessions".
-A session stores data specific to a given client that can only be accessed by that client (based on the client's cookie identity).
+As you might imagine, this might get unwieldy after a while, in large stylesheets.  This is one example of a place where using a CSS pre-processer or extension language like Less or Sass comes in handy.
 
-Now add this line to your `require` list:
+## Less and Sass
 
-```node
-var session = require('express-session');
-```
+These are CSS pre-processor languages that allow you to use simpler and more manageable syntax that converts to CSS.  Before class, please [install Sass](http://sass-lang.com/install), and read about [Sass features](http://sass-lang.com/guide).  We'll cover some Sass usage in class.
 
-And this line at the end of your `app.use` list:
+This class is going to focus just on Sass, because it's more widely used than Less, and the two have very similar features with slightly different syntax.  If you're interested in some comparisons between the two, the following articles are optional:
+- [Intro to Less + comparison to Sass](http://www.smashingmagazine.com/2011/09/an-introduction-to-less-and-comparison-to-sass/)
+- [Sass vs Less](https://css-tricks.com/sass-vs-less/)
+- [Another Sass vs Less](http://www.hongkiat.com/blog/sass-vs-less/)
 
-```node
-app.use(session({
-	secret: 'secret',
-	resave: false,
-	saveUninitialized: true
-}));
+Once you've installed Sass, you can create .sass or .scss stylesheets according to their syntax (the difference is, SCSS uses {} and semicolons, Sass relies on indentation).  Then, you can create your CSS stylesheets:
```

Comment:
For sanity and consistency's sake, I would pick a specific syntax (my preference is scss).